### ISSUE TITLE
How to place objects in the same place always

### ISSUE THREAD

**Issue by omarojo** [gao0122/ARKit-Example-by-Apple#4]
I was wondering if there is a way to place objects in a textured poster in the wall.. or QR code print (aka, MARKERS).
ARkit detects planes and allows you to place object in those planes. But I dont care about those planes... I want all the users of my app to see the same object in the same place in the room, maybe by the use of a marker (QRCode, ImagePhoto) or something placed in the physical world so the app knows where to place the object.

Do you know if this is possible with ARKit ?

**Comment by gungorbasa** [comment]
In the simplest sense, you can try something like pattern matching. Just look for the pattern in the image or you can use qrcode as well. When you recognize the qr code then put your object there.

**Comment by omarojo** [comment]
Yeah but then keep the orientation of the object according to the qrcode/marker plane while moving the camera, thats a whole new deal.

So no support for this from ARKit right ?

**Comment by washaq** [comment]
@omarojo did you find anything on it?

**Comment by omarojo** [comment]
Negative :(

**Comment by washaq** [comment]
:|

**Comment by ezanalemma** [comment]
I'm searching for the solution too
